modules/registry-configuring-storage-aws-user-infra.adoc

```diff
@@ -21,7 +22,7 @@ procedure.
 Use the following procedure if the Registry Operator cannot create an S3 bucket
 and automatically configure storage.
 
-. Create a S3 bucket:
+. Create an S3 bucket:
 +
 ----
 $ oc create configs.imageregistry.operator.openshift.io
```

Comment:
I do not think this command is used for creating an s3. s3 should be created manually by user in this case from aws cli or web console, isn't it?
And even this command can not be executed successfully.
`$ oc create configs.imageregistry.operator.openshift.io
Error: required flag(s) "filename" not set`

Reply:
Brandi says that it's fine to remove this command.